src/output.rs

```diff
@@ -226,7 +230,7 @@ pub fn output_covdir(results: CovResultIter, output_file: Option<&Path>) {
 
         prev_stats.borrow_mut().files.push(CDFileStats::new(
             path.file_name().unwrap().to_str().unwrap().to_string(),
-            result.lines,
+            result.lines.clone(),
```

Comment:
Are this `clone` and the others you introduced in this file necessary? It'd be nice to be able to avoid to allocate memory here

Reply:
They are reequired, i checked with clippy

Reply:
Maybe there's a refactoring that you could do to make them unnecessary?
Anyway, before worrying too much, let's try to run the benchmarks before/after your change.

Reply:
The benchmarks don't show any performance degradation.